src/nvme/fabrics.c

```diff
@@ -350,7 +347,7 @@ static int inet_pton_with_scope(int af, const char *src, const char * trsvcid,
 			ret = inet6_pton(src, port, addr);
 		break;
 	default:
-		nvme_msg(LOG_ERR, "unexpected address family %d\n", af);
+		ret = -EINVAL;
```

Comment:
I don't mind these error handling cleanups in inet*(). But it would be better to go into a separate patch.